tfprotov5/tftypes/value.go

```diff
@@ -609,13 +609,19 @@ func (val Value) IsFullyKnown() bool {
 	case primitive:
 		return true
 	case List, Set, Tuple:
+		if val.value == nil {
```

Comment:
would it not be sufficient to do this check once right before `switch {` ?

Reply:
Good call.